data/json/construction.json

```diff
@@ -3176,6 +3176,17 @@
     "pre_special": "check_empty",
     "post_special": "done_appliance"
   },
+  {
+    "type": "construction",
+    "id": "app_chain_hoist",
+    "group": "place_chain_hoist",
+    "category": "APPLIANCE",
+    "required_skills": [ [ "fabrication", 3 ] ],
+    "time": "5 m",
+    "components": [ [ [ "chain_hoist", 1 ] ] ],
+    "pre_special": "check_empty",
```

Comment:
```suggestion
    "pre_special": "check_support",
```
So you can only place it under a roof. Probably need some tools too.

Reply:
Oh, that's a great idea. I'll try it out.

Reply:
Implemented